spec/support/vcr.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "vcr"
+
+ENV["ATRIUM_API_KEY"]   = ENV.fetch("ATRIUM_API_KEY",   "ATRIUM_API_KEY")
+ENV["ATRIUM_CLIENT_ID"] = ENV.fetch("ATRIUM_CLIENT_ID", "ATRIUM_CLIENT_ID")
+
+VCR.configure do |config|
+  config.cassette_library_dir = "spec/cassettes"
+  config.hook_into :webmock
+
+  config.configure_rspec_metadata!
+
+  config.filter_sensitive_data("ATRIUM_API_KEY")   { ENV["ATRIUM_API_KEY"] }
+  config.filter_sensitive_data("ATRIUM_CLIENT_ID") { ENV["ATRIUM_CLIENT_ID"] }
+end
+
+RSpec.configure do |config|
+  # Add VCR to all tests
+  config.around(:each) do |example|
+    options = example.metadata[:vcr] || {}
+
+    if options[:record] == :skip
+      VCR.turned_off(&example)
+    else
+      name = example
```

Comment:
Not really, this VCR hooks into webmock and intercepts all network requests. This is so you don't have to mock out the client, and can actually test your code. The rest of the specs need to be updated to use VCR. As it is, the completely mocked out, and untested, client contains bugs due to this incomplete usage.

Most of the specs test the mocks and not the code. Which is how, and why, I discovered this bug: https://github.com/mxenabled/atrium-ruby/pull/31/commits/90211ae051273525fc0da10d1dcf1afa231a3193